packages/core/src/rest/api/ZosmfRestClient.ts

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
Same question as above - curious about the "rest/api" folder

Reply:
fixed